src/online/online_player_profile.cpp

```diff
@@ -403,7 +403,8 @@ namespace Online
                     }
                     else if(to_notify.size() > 3)
                     {
-                        message = _("%d friends are now online.",
+                        message = _P("%d friends are now online.",
+                                     "%d friends are now online.",
```

Comment:
There are 3 cases above that if which handle the cases 1, 2, and 3.

It might be a bit better to have the singular case be a real singular, but it isn't really used.

There are some languages (eg Gaelic; see #1852) where you have different plurals so the will have wrong translations.